src/Commands/ImportDump.php

```diff
@@ -246,4 +333,39 @@ public function handle()
             $this->info('Import aborted');
         }
     }
+
+    /**
+     * Returns a list from either all or from the specified connection name dumps.
+     *
+     * @param Collection $matchingFiles
+     * @param string|null $connectionName
+     * @return Collection
+     */
+    public function getConnectionFiles(Collection $matchingFiles, ?string $connectionName = null): Collection
+    {
+        $sortedFiles = $matchingFiles->sortByDesc('dateTime')->groupBy('connection');
+
+        if ($sortedFiles->count() == 1) {
+            $connectionName = Arr::first($sortedFiles->keys()->sort()->toArray());
```

Comment:
how can the connection be replaced anyway, when its a given parameter

Reply:
see https://trello.com/c/6ea2rj48/1919-protector-does-ignoreconnectionfilter-make-sense